### PR TITLE
Send/Receive messages

### PR DESCRIPTION
**Pull request by nakajima**
Extracted from https://github.com/xmtp/xmtp-ios/pull/6.
Relies on https://github.com/xmtp/xmtp-ios/pull/9.

### CODE REVIEW

File: Sources/XMTP/Client.swift [hunk 1]
```diff
@@ -38,7 +38,27 @@ class Client {
...
+
+			var apiClient = apiClient
+			apiClient.setAuthToken(authToken)
+
+			do {
+				try await apiClient.publish(envelopes: [
+					Envelope(topic: .userPrivateStoreKeyBundle(account.address), timestamp: Date(), message: try encryptedKeys.serializedData()),
+				])
+			} catch {
+				// Unable to save keys
```
Comment (elisealix22) [inline Sources/XMTP/Client.swift:58]: Maybe worth bubbling this up or returning nil if it breaks things? Or can we move on without it?

File: Sources/XMTP/Codecs/TextCodec.swift [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+//  TextCodec.swift
+//
+//
+//  Created by Pat Nakajima on 11/28/22.
+//
+
+import Foundation
+import XMTPProto
+
+let ContentTypeText = ContentTypeID(authorityID: "xmtp.org", typeID: "text", versionMajor: 1, versionMinor: 0)
```
Comment (elisealix22) [inline Sources/XMTP/Codecs/TextCodec.swift:11]: 🔥

**Review by elisealix22 — APPROVED** [review]
Wait, what the heck? This is the whole SDK  😆 . Just one question / suggestion on the saving of keys in creating the client. Otherwise, this is epic.

File: Sources/XMTP/Client.swift (hunk 1 above)
Comment (nakajima) [reply Sources/XMTP/Client.swift]: Ya, I agree. I've removed this try/catch.

**Pull request merged**